md-commons/src/main/java/com/wipro/ats/bdre/md/beans/table/Process.java

```diff
@@ -35,6 +35,54 @@
     private String tableEditTS;
     private Boolean deleteFlag;
     private Integer workflowId;
+    private Integer permissionTypeByOthersAccessId;
+    private Integer permissionTypeByUserAccessId;
+    private Integer permissionTypeByGroupAccessId;
+    private String userName;
+    private Integer ownerRoleId;
+    public Integer getOwnerRoleId() {
+        return ownerRoleId;
+    }
+
+    public void setOwnerRoleId(Integer ownerRoleId) {
+        this.ownerRoleId = ownerRoleId;
+    }
+
+
+
+    public String getUserName() {
+        return userName;
+    }
+
+    public void setUserName(String userName) {
+        this.userName = userName;
+    }
+
+
+
+    public Integer getPermissionTypeByOthersAccessId() {
+        return permissionTypeByOthersAccessId;
+    }
+
+    public void setPermissionTypeByOthersAccessId(Integer permissionTypeByOthersAccessId) {
+        this.permissionTypeByOthersAccessId = permissionTypeByOthersAccessId;
+    }
+
+    public Integer getPermissionTypeByUserAccessId() {
+        return permissionTypeByUserAccessId;
+    }
+
+    public void setPermissionTypeByUserAccessId(Integer permissionTypeByUserAccessId) {
+        this.permissionTypeByUserAccessId = permissionTypeByUserAccessId;
+    }
+
+    public Integer getPermissionTypeByGroupAccessId() {
+        return permissionTypeByGroupAccessId;
+    }
+
+    public void setPermissionTypeByGroupAccessId(Integer permissionTypeByGroupAccessId) {
+        this.permissionTypeByGroupAccessId = permissionTypeByGroupAccessId;
+    }
     @NotNull
     @Size(min = 1, max = 256)
     private String description;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)